packages/quantic/cypress/integration/facets/facet-common-selectors.ts

```diff
@@ -0,0 +1,31 @@
+import {ComponentSelector, CypressSelector} from '../common-selectors';
+
+export interface BaseFacetSelector extends ComponentSelector {
+  label: CypressSelector;
+  values: CypressSelector;
+  clearButton: CypressSelector;
+  valueLabel: CypressSelector;
+  facetValueLabelAtIndex: (
+    index: number
+  ) => Cypress.Chainable<JQuery<HTMLElement>>;
```

Comment:
```suggestion
  ) => CypressSelector;
```

Reply:
`CypressSelector` includes the function arguments, so I can't use it like you suggest. Also, I can't define generics for the selectors taking arguments, Typescript does not support that.

I can change `CypressSelector` to not include the function arguments. It would be a little longer to write, but it would also work for selectors with arguments.

Reply:
Meh ¯\_(ツ)_/¯ up to you. I really like how much cleaner things look now.